key_management/src/keystore.rs

```diff
@@ -1,10 +1,17 @@
 // Copyright 2020 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
+extern crate serde_json;
+
 use super::errors::Error;
 use crypto::SignatureType;
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
+use std::fs::File;
+use std::fs::OpenOptions;
+use std::io::BufReader;
+
+const KEYSTORE_LOCATION: &str = "keystore.json";
```

Comment:
should have this point to a folder in the data dir directory from the config. This will just place a file into whatever directory the binary is run in